src/framework/uicomponents/qml/MuseScore/UiComponents/internal/StyledMenuItem.qml

```diff
@@ -92,7 +92,7 @@ ListItemBlank {
             text += " " + qsTrc("ui", "Menu")
         }
 
-        return Utils.removeAmpersands(text)
```

Comment:
I'm not sure that we get text without mnemonics here.

Reply:
`text` is based on `itemPrv.title`, which contains certainly no mnemonic ampersands. The other strings that are appended to it also certainly don't contain ampersands (and for the translators, I think it is clear enough that they should not add ampersands here).